src/main/java/spring/data/access/service/ProductDao.java

```diff
@@ -0,0 +1,26 @@
+package spring.data.access.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+import spring.data.access.repository.JDBCTemplateProduct;
+import spring.data.access.entity.Product;
+
+@Component
+@Scope("prototype")
+public class ProductDao {
```

Comment:
its definitely Service not ProductDao
pls rename

Reply:
renamed